frontend/src/components/common/ReleaseNotes/UpdatePopup.tsx

```diff
@@ -2,11 +2,52 @@ import { Button, Snackbar } from '@material-ui/core';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 
-function UpdatePopup(props: { releaseDownloadURL: string }) {
+function UpdatePopup(props: {
+  releaseDownloadURL?: string | null;
+  fetchingRelease?: boolean;
+  releaseFetchFailed?: boolean;
+}) {
   const [show, setShow] = React.useState(true);
-  const { releaseDownloadURL } = props;
+  const [releaseFetchShow, setReleaseFetchShow] = React.useState(true);
+  const { releaseDownloadURL, fetchingRelease, releaseFetchFailed } = props;
   const { t } = useTranslation('frequent');
 
+  if (fetchingRelease) {
+    return (
+      <Snackbar
+        open={releaseFetchShow}
+        message={t('release|fetching release...')}
+        ContentProps={{
+          'aria-describedby': t('release|fetching release message'),
+        }}
+        action={
+          <React.Fragment>
+            <Button color="primary" onClick={() => setReleaseFetchShow(false)}>
+              {t('frequent|Skip')}
+            </Button>
```

Comment:
This doesn't skip fetching the release, just doesn't show this snackbar. More details in the final review comment.

Reply:
fixed